utils/ANLEra_auto_working.cfg

```diff
@@ -327,3 +327,95 @@
         [/redraw]
     [/event]
 #enddef
+
+# ------------------------------------------------------
+# Automatic Hunting
+# ------------------------------------------------------
+
+#define AUTO_HUNTING TYPES
+    [event]
+        name=side turn
```

Comment:
If you have an way to reproduce #2, this would be the place to try side turn refresh.